package.json

```diff
@@ -27,12 +27,12 @@
     "@typescript-eslint/eslint-plugin": "^6.2.1",
     "@typescript-eslint/parser": "^6.2.1",
     "eslint": "^8.46.0",
-    "eslint-config-prettier": "^8.9.0",
+    "eslint-config-prettier": "^9.0.0",
```

Comment:
See below for why I bumped this too. Ended up being unrelated, but it's a good thing to update anyway. 😁